crates/net/rpc-types/src/eth/call.rs

```diff
@@ -1,4 +1,4 @@
-use reth_primitives::{rpc::transaction::eip2930::AccessListItem, Address, Bytes, U256};
+use reth_primitives::{rpc::transaction::eip2930::AccessListItem, Address, Bytes, U128, U256};
```

Comment:
all changes to `rpc-types` are unrelated to the issue, which is only for txpool.

we probably need them as well, because we need to do some conversion from pool/disk transaction type to RPC type.

Could you please submit them separately?